stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1600,7 +1600,13 @@ internal constructor(
                     interceptor.interceptMessage(channelType, channelId, message.data(), isRetrying)
                 } else message
             }.flatMapSuspend { newMessage ->
-                api.sendMessage(channelType, channelId, newMessage)
+                api.sendMessage(
+                    channelType = channelType,
+                    channelId = channelId,
+                    message = newMessage,
+                    skipPushNotification = message.skipPushNotification,
+                    skipEnrichUrl = message.skipEnrichUrl,
```

Comment:
```suggestion
                    skipPushNotification = newMessage.skipPushNotification,
                    skipEnrichUrl = newMessage.skipEnrichUrl,
```
I assume it should contain the same data, but it could be confusing if we use different entities in this method

Reply:
Good catch, probably an autocomplete mistake.

Reply:
Updated